frontend/src/components/city_table/city_table.tsx

```diff
@@ -54,49 +67,24 @@ export const CityTable: React.FC = () => {
     }
   };
 
-  const [selected, setSelected] = useState<City>();
-  const [searchString, setSearchString] = useState("");
-
   const handleOnSelect = (item: City) => {
     setCityId(item.id);
     setSearchString("");
   };
 
-  const [rowOpen, setRowOpen] = useState<IIsOpen>({
```

Comment:
why has this whole section been removed?

it's very important for running the whole application

Reply:
Hasn't been removed, just moved to line 29 to keep all the State together

Reply:
oops - total schoolboy from me

looks good